docker/Dockerfile

```
@@ -13,7 +13,7 @@ RUN mkdir -p /src
 COPY package.json /src
 
 # Install app dependencies
-RUN apk add --no-cache sed nodejs && \
+RUN apk add --no-cache npm && \
```

sed still required at this step.

```suggestion
RUN apk add --no-cache sed npm && \
```

---

What is sed used for? I just built the image without sed (exactly as it is in this version, from this branch) and everything seems to work fine.

---

Ok. I see in the git history, that `sed` was required before, but removed, and no more required.

But the README.md also ask to install `sed`. I think we can also remove it fro there.

---

just pushed a change